css/css-scroll-snap/scroll-target-snap-001.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<title>#target and snap position with snapping on (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#choosing'>
+<link rel='match' href='scroll-target-001-ref.html'>
+
```

Comment:
An assert meta would be welcome, similarly to scroll-target-001

Reply:
Done.